recipes/roboticstoolbox-python/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "roboticstoolbox-python" %}
+{% set version = "0.9.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/roboticstoolbox-python-{{ version }}.tar.gz
+  sha256: efa9165125a07156097db5f6b6fc4990f7f8615d6a202c89f292a808da00aa26
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Assuming it's not actually using the compiler:
```suggestion
```

Reply:
It uses C extensions, so I'm pretty sure it needs this?

Reply:
See https://github.com/petercorke/robotics-toolbox-python/blob/0e97148c186f83f70d36a0adeff23c0bfc872fb4/setup.py#L69